assets/js/base/components/checkbox-list/index.tsx

```diff
@@ -11,6 +10,21 @@ import classNames from 'classnames';
  */
 import './style.scss';
 
+interface CheckboxListOptions {
+	label: React.ReactNode;
+	value: string;
+}
+
+interface CheckboxListProps {
```

Comment:
I'd check which props are required and which are optional. This type makes everything required, while the previous type had almost everything as optional.

I'd definitely check the code to see what's really required by the component. Because things like `className` are often not among them.